### ISSUE TITLE
`buildables` block type?

### ISSUE THREAD

**Issue by korrat** [divnix/std#378]
Both `installables` and `runnables` block types provide a `build` action. However, both block types also provide further actions. This makes them less than ideal for some use cases, such as checks or, in the case of Rust projects, a separate derivation for `cargoArtifacts`.

A simple `buildable` block type may be helpful here. This block type would only provide the `build` action. Projects could use it to provide a set of dependencies that other blocks of the cell could use.

Alternatively, both the `anything` and `functions` block types could be used for this purpose. In the case of checks, however, that means that checks can no longer be run individually.

**Comment by blaggacao** [comment]
> Both installables and runnables block types provide a build action. However, both block types also provide further actions. This makes them less than ideal for some use cases, such as checks or, in the case of Rust projects, a separate derivation for cargoArtifacts.

Agreed. This is a long-standing semantic mismatch that I just didn't bother to resolve.

I think a low-risk/low-maint and backward-compatible approach would be to perpetuate, but alleviate this issue by just adding your suggested buildable block type.
